english/little-me/Apple/Apple.md

```diff
@@ -0,0 +1,10 @@
+Thank you so much.
+Now that I am full I can rewind the time.
+
+Do you like to see my powers?
+
+You Choosed To Say:
+
+[Yes](https://github.com/udacity/create-your-own-adventure/tree/master/english/little-me/little-me.md)
+
+[No](https://github.com/udacity/create-your-own-adventure/tree/master/english/)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.